services/QuillLMS/app/workers/copy_concept_results_to_student_responses_worker.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CopyConceptResultsToStudentResponsesWorker
```

Comment:
What's the case for the job looping through a batch of `concept_ids` vs. 1 job per `concept_id`? This isn't idempotent, so wouldn't we end up with duplicate `student_responses` if this job errors midway through?

Reply:
It may be reasonable to rework this from a batch executor to an individual executor just to keep things simple.  It sort of felt to me like putting fewer jobs on the queue might be a benefit, but on second thought I'm not totally sure that's true.

As for idempotency, `StudentResponse.create_from_concept_result` itself should be idempotent (there are tests around that), so if some items in a batch fail, rerunning the whole batch should be safe.